### PR TITLE
LPS-196939 Modify expanded keys to set just one key

### PR DESCRIPTION
**Pull request by HenriquePereiraa**
I made a modification to put only the key of the selected folder in the expanded keys state so that it will only be open when entering the Object Model Builder.

[Ticket link](https://liferay.atlassian.net/browse/LPS-196939)

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.tsx [hunk 1]
```diff
@@ -55,10 +55,14 @@ export default function LeftSidebar({setShowModal}: LeftSidebarProps) {
 			};
 		});
 
-		setExpandedKeys(new Set(keys));
+		const selectedObjectFolderKey = keys.filter(
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.tsx:58]: @HenriquePereiraa If you're retrieving just one item, it's better to use a find than a filter then. Can you change it, please? Thanks!

**Review by naraandrade — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.tsx (hunk 1 above)
Comment (HenriquePereiraa) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.tsx]: Ah, I get it!

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:stop

**Comment by naraandrade** [comment]
@HenriquePereiraa SF failure, can you please take a look? Thanks!

**Comment by HenriquePereiraa** [comment]
> @HenriquePereiraa SF failure, can you please take a look? Thanks!

Ok @naraandrade , I'll check it out!

**Comment by HenriquePereiraa** [comment]
ci:test:relevant

**Comment by HenriquePereiraa** [comment]
ci:test:sf

**Comment by HenriquePereiraa** [comment]
@naraandrade, the ci relevant error doesn't seem to have anything to do with my changes.

**Comment by naraandrade** [comment]
ci:forward:force

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by naraandrade** [comment]
Resent here: https://github.com/brianchandotcom/liferay-portal/pull/142468

**Pull request closed without merging**